README.md

```diff
@@ -142,6 +142,19 @@ You can address the error `ssh: handshake failed: knownhosts: key is unknown ` w
 - Calling `ssh-keyscan -H github.com >> ~/.ssh/known_hosts` prior to pushing your vergo tag to introduce github to your known hosts.
 - Calling `vergo` with the `--disable-strict-host-check` flag. This should only be used on CI where known hosts are not cached.
 
+## Using GITHUB_TOKEN inside GitHub Actions
+
+Vergo with prefer to use Token Bearer Authentication when running inside a GitHub Action/Workflow.
+Please ensure that the GITHUB_TOKEN environment variable is set with the `${{ secrets.GITHUB_TOKEN }}` in order to push to the current repository.
```

Comment:
Is this line specific to using within a GH action?

Reply:
Added a little more detail here if it makes sense